chainer/functions/array/broadcast.py

```diff
@@ -54,11 +54,11 @@ def broadcast(*args):
     """Broadcast given variables.
 
     Args:
-      args (Variables): Variables to be broadcasted.
+        args (Variables): Variables to be broadcasted.
 
     Returns:
-      ``tuple``: Tuple of :class:`~chainer.Variable` objects which are
-          broadcasted from given arguments.
+        tuple: Tuple of :class:`~chainer.Variable` objects which are
```

Comment:
`tuple` refers a method which makes a tuple object, not a type itself. Is that OK?
https://docs.python.org/2/library/functions.html#tuple